examples/react-server/vite.config.ts

```diff
@@ -318,6 +316,9 @@ function vitePluginServerAction(): PluginOption {
       }
       const ast = await parseAstAsync(code);
       tinyassert(this.environment);
+      if (id.startsWith(this.environment.config.root)) {
+        id = id.slice(this.environment.config.root.length);
+      }
```

Comment:
It looks like there's no duplicate module when using project root relative e.g. `import("/src/routes/_action.tsx")` instead of full path.

Reply:
We should be normalizing reference id anyway, so let's merge this for now to unblock ecosystem-ci.